tests/cypress/support/commands.js

```diff
@@ -253,7 +253,7 @@ Cypress.Commands.add('CheckGrcMainPage', () => {
 
 Cypress.Commands.add('FromGRCToCreatePolicyPage', () => {
   cy.get('#create-policy').should('exist')
-  cy.get('#create-policy').click()
+  cy.get('#create-policy').should('have.attr', 'aria-disabled', 'false').click()
```

Comment:
I don't think this is going to work. The `should` I think returns the attribute itself. We'll see what the test does, but I think you'll need to re-get the button to click on it after checking the attribute.

https://docs.cypress.io/api/commands/should#Yields

Reply:
Yeah, it reads like it. But I have tested it locally and it worked. Still a mystery.

Reply:
I guess since you've specified a value, `false`, then it returns the element since you've already checked the value. That's nice--and good to know! 🙂

Reply:
makes sense